proto/docs/p4runtime.md

```diff
@@ -176,11 +176,17 @@ switch(p4_type(match.field_id())) {
   case EXACT:
     assert(match.has_exact() && !match.exact().value().empty())
   case TERNARY:
+    Integer value, mask;
     assert(match.has_ternary())
     assert(!match.ternary().value().empty() && !match.ternary().mask().empty())
+    assert(parseInteger(match.ternary().value(), &value))
+    assert(parseInteger(match.ternary().mask(), &mask))
+    assert(value == value & mask)
```

Comment:
Doesn't == have higher precedence than bitwise & ?

Reply:
Thanks for spotting this; I mistakenly removed the parenthesis when I made the `parseInteger` change.